docs/results/reproduction-results.md

```
@@ -264,12 +264,16 @@ make dev
 ollama pull gemma3:27b-it-qat
 ollama pull qwen3-embedding:8b
 
-# 3. Create paper splits (58/43/41)
-uv run python scripts/create_paper_split.py --verify
+	# 3. Create paper splits (58/43/41)
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)